pkg/bpf/map.go

```diff
@@ -643,7 +643,7 @@ func (m *Map) DeleteWithErrno(key MapKey) (error, syscall.Errno) {
 	_, errno = deleteElement(m.fd, key.GetKeyPtr())
 
 	if errno != 0 {
-		err = fmt.Errorf("Unable to delete element from map: %s", err)
+		err = fmt.Errorf("Unable to delete element from map: %s", m.name)
```

Comment:
I think this may have been a typo and should have said ``errno``. I think it would be useful to print both, ``m.name`` and the actual ``errno`` along with it.

Reply:
Thanks @borkmann, Updated and rebased the PR